var/spack/repos/builtin/packages/likwid/package.py

```diff
@@ -87,8 +87,19 @@ def install(self, spec, prefix):
                     prefix,
                     'config.mk')
 
-        filter_file('^INSTALL_CHOWN.*',
-                    'INSTALL_CHOWN = -o $(USER)',
+# FIXME: once https://github.com/spack/spack/issues/4432 is resolved, install
+# as root by default and remove this
+        filter_file('^ACCESSMODE .*',
+                    'ACCESSMODE = ' +
+                    'perf_event',
+                    'config.mk')
+        filter_file('^BUILDFREQ .*',
+                    'BUILDFREQ = ' +
+                    'false',
+                    'config.mk')
+        filter_file('^BUILDDAEMON .*',
+                    'BUILDDAEMON = ' +
+                    'false',
```

Comment:
Can you put these strings on the same line instead of breaking the string in the middle?